remotespark/utils/configuration.py

```diff
@@ -144,6 +144,12 @@ def logging_config():
         }
     }
 
+@_override
+def events_config():
+    return {
+        "handler": "remotespark.utils.jupytereventfilehandler"
```

Comment:
* Don't put the string `Jupyter` in any file names (remember, these events have nothing to do with Jupyter)

* Remember that the default handler is not a "file" handler